### PR TITLE
ref#9957 annotation description added

### PR DESCRIPTION
**Pull request by NastyBoget**

### CODE REVIEW

File: dedoc/api/static/html_rus/format_description.html [hunk 1]
```diff
@@ -91,11 +91,40 @@ <h3 id="TreeNode"> TreeNode. Древовидная структура доку
...
+        Для документов в формате docx отступ измеряется в двадцатых пункта (1/1440 дюйма). </li>
+    <li><strong> ItalicAnnotation </strong> : выделен ли текст курсивом. Значение поля name = "italic".
+        Значением поля value может быть "True", если текст выделен курсивом и "False" иначе. </li>
+    <li><strong> SizeAnnotation </strong> : размер шрифта в пунктах (1/72 дюйма). Значение поля name = "size".
+        Значением поля value может быть любое действительное число, сконвертированное в строку.</li>
+    <li><strong> StyleAnnotation </strong> : название стиля, примененного к тексту, например, "heading 1".
+        Значение поля name = "style".
+        Значением поля value может быть любая строка.</li>
+    <li><strong> Table Annotation </strong> : является ли строка ссылкой на таблицу. Значение поля name = "table".
+        Значение поля value = "table " + уникальный идентификатор таблицы. </li>
```
Comment (IlyaKozlov) [inline dedoc/api/static/html_rus/format_description.html:123]: name it is not "table" + table_name but only table name.

File: dedoc/api/static/html_eng/format_description.html [hunk 2]
```diff
@@ -87,11 +87,40 @@ <h4 id="Annotation"> Annotation</h4>
...
+        For docx documents, the indent is measured in twentieths of a point (1/1440 of an inch). </li>
+    <li><strong> ItalicAnnotation </strong> : italic text. The value of "name" field = "italic".
+        "value" may be "True", if the text is in italics and "False" otherwise. </li>
+    <li><strong> SizeAnnotation </strong> : font size in points (1/72 of an inch). The value of "name" field = "size".
+         "value" may be any real number converted to a string. </li>
+    <li><strong> StyleAnnotation </strong> : the name of the style applied to the text, for example, "heading 1".
+        The value of "name" field = "style".
+        "value" may be any string. </li>
+    <li><strong> Table Annotation </strong> : table reference. The value of "name" field = "table".
+        "value" field = "table " + unique identifier of the table. </li>
```
Comment (IlyaKozlov) [inline dedoc/api/static/html_eng/format_description.html:119]: name it is not "table" + table_name but only table name.

File: dedoc/api/static/html_rus/format_description.html (hunk 1 above)
Comment (NastyBoget) [reply dedoc/api/static/html_rus/format_description.html]: done

File: dedoc/api/static/html_eng/format_description.html (hunk 2 above)
Comment (NastyBoget) [reply dedoc/api/static/html_eng/format_description.html]: done

**Pull request merged**